README.md

```diff
@@ -1,10 +1,229 @@
+</picture>
+<p>
+
+[Coinbase](https://www.coinbase.com/) was founded by Brian Armstrong and Fred Ehrsam in 2012 with the goal to "increase economic freedom in the world". Coinbase Pro provides access to trading Crypto for clients in over [100 countries](https://www.coinbase.com/places) with no minimum deposit. Coinbase Pro also provides a self-hosted Crypto wallet, a Visa debit rewards card, and Bitcoin collateral-backed lines of credit.
+
+## Using the Brokerage Plugin
+  
+### Deploying Coinbase Pro with VSCode User Interace
+
+  You can deploy using a visual interface in the QuantConnect Cloud. For more information see the [QuantConnect-CoinbasePro Integration Page](https://www.quantconnect.com/brokerages/coinbasepro).
+
+  In the QuantConnect Cloud Platform you can harness the QuantConnect Live Data Feed. For most users this is substantially cheaper and easier than self-hosting.
+  
+### Deploying Coinbase Pro with LEAN CLI
+
+Follow these steps to start local live trading with the Coinbase Pro brokerage:
+
+1.  Open a terminal in your [CLI root directory](https://www.quantconnect.com/docs/v2/lean-cli/initialization/directory-structure#02-lean-init).
+2.  Run lean live "`<projectName>`" to start a live deployment wizard for the project in ./`<projectName>` and then enter the brokerage number.
+
+    ```
+    $ lean live 'My Project'
+     
+    Select a brokerage:
+    1. Paper Trading
+    2. Interactive Brokers
+    3. Tradier
+    4. OANDA
+    5. Bitfinex
+    6. Coinbase Pro
+    7. Binance
+    8. Zerodha
+    9. Samco
+    10. Terminal Link
+    11. Atreyu
+    12. Trading Technologies
+    13. Kraken
+    14. FTX
+    Enter an option: 
+    ```
+  
+3.  Enter the number of the organization that has a subscription for the Coinbase Pro module.
+
+    ```
+    $ lean live "My Project"
+
+    Select the organization with the Coinbase Pro module subscription:
+    1. Organization 1
+    2. Organization 2
+    3. Organization 3
+       Enter an option: 1
+    ```
+
+4.  Enter the environment to use.
+
+    ```
+    $ lean live "My Project"
+
+    Use sandbox? (live, paper): live
+    ```
+
+5.  Enter your API key, API secret, and passphrase.
+
+    ``` 
+    $ lean live "My Project"
+
+    API key: 6d3ef5ca2d2fa52e4ee55624b0471261
+    API secret: ****************************************************************************************
+    Passphrase: ****************
+    ```
+
+    To create new API credentials, see the [API settings page](https://pro.coinbase.com/profile/api) on the Coinbase Pro website.
+
+
+6.  Enter the number of the data feed to use and then follow the steps required for the data connection.
+
+    ``` 
+    $ lean live 'My Project'
+
+    Select a data feed:
+    1. Interactive Brokers
+    2. Tradier
+    3. Oanda
+    4. Bitfinex
+    5. Coinbase Pro
+    6. Binance
+    7. Zerodha
+    8. Samco
+    9. Terminal Link
+    10. Trading Technologies
+    11. Kraken
+    12. FTX
+    13. IQFeed
```

Comment:
Should IQFeed be here?

Reply:
I think so because it's a valid option, and `lean-cli` will display it.